### DISCUSSION TITLE
Upgraded version 5.0.0-dev.2060 create interface prompt "Method not found: the Windows UI. Xaml. UIElement. The set_Name (System. String)"

### DISCUSSION THREAD

**Discussion by Sharktao** [unoplatform/uno#13327] [Q&A] [answered]
1.TargetFramework(.net7.0) + Uno(4.9.17), the software runs normally;

2.TargetFramework(.net7.0) + Uno(5.0.0-dev.2060), the software runs abnormally;

3. Can you provide release notes.

**Comment by jeromelaban** [comment]
Thanks for the update. You're likely using an additional library that is not updated to Uno 5 and is still using Uno 4 packages. Make sure to update all your references, and if you're using other Uno Platform packages, Uno 5 compatible versions are available.

We're not providing release notes for prerelease packages at this time.

**Reply by Sharktao** [reply]
1.Netstandard2.0 was not found in the dependencies for version 5.0.0-dev.2060,;
2.How to deal with the xmal file developed on the basis of netstandard2.0?

**Reply by jeromelaban** [reply, marked as answer]
`netstandard2.0` is not supported anymore to build uno libraries in Uno 5.0+. You'll need to update to .NET 7 or later.

**Comment by Sharktao** [comment]
netstandard2.0  update to .NET 7;

![9DC59E73-FEB4-487d-86B4-20989B5CD357](https://github.com/unoplatform/uno/assets/43270517/1a23a46b-a5c4-4b82-afad-7467f192a8b8)
![54CAD55E-633C-4e2d-8068-636C186E91B1](https://github.com/unoplatform/uno/assets/43270517/a1189f1a-3a73-4f5e-acf8-abb0210306e1)

x:Name objects cannot be used in cs files,  What I need to do?

**Reply by jeromelaban** [reply]
This looks like an generation error. Are you able to build your app and [provide a binlog](https://platform.uno/docs/articles/uno-builds-troubleshooting.html)?

**Reply by jeromelaban** [reply]
You can follow the link I posted above to create a binlog and attach it here.

**Reply by Sharktao** [reply]
An error was located. The X:Bind + method ;

![微信图片_20230828160434](https://github.com/unoplatform/uno/assets/43270517/b4a5b875-8a74-4806-966c-1f787992817e)

**Reply by Sharktao** [reply]
Thank you for your help!
